src/gpuarray/config.h

```diff
@@ -1,6 +1,8 @@
 #ifndef GPUARRAY_CONFIG
 #define GPUARRAY_CONFIG
 
+/* The following included file should have been generated by CMake. */
+#include <gpuarray/abi_version.h>
 #define GPUARRAY_API_VERSION 0
```

Comment:
I would remove this line, as it should be in the included file. Otherwise, it is an error and not having it with a wrong value should help with that.